src/components/AttachmentManager/AttachmentManager.js

```diff
@@ -1,13 +1,12 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import { DragDropContext, DragSource, DropTarget } from 'react-dnd'
-import HTML5Backend from 'react-dnd-html5-backend'
-import path from 'path'
 import { isEmpty, filter } from 'lodash/fp'
-import { bgImageStyle } from 'util/index'
-import Loading from 'components/Loading'
+import path from 'path'
+import PropTypes from 'prop-types'
+import React, { useRef } from 'react'
+import { useDrag, useDrop } from 'react-dnd'
```

Comment:
This is all just updating react-dnd to its new APIs?

Reply:
yeah